modules/core/04-channel/v2/keeper/msg_server_test.go

```diff
@@ -0,0 +1,112 @@
+package keeper_test
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	clienttypes "github.com/cosmos/ibc-go/v9/modules/core/02-client/types"
+	channeltypesv1 "github.com/cosmos/ibc-go/v9/modules/core/04-channel/types"
+	channeltypesv2 "github.com/cosmos/ibc-go/v9/modules/core/04-channel/v2/types"
+	ibctesting "github.com/cosmos/ibc-go/v9/testing"
+	"github.com/cosmos/ibc-go/v9/testing/mock"
+	mockv2 "github.com/cosmos/ibc-go/v9/testing/mock/v2"
+)
+
+func (suite *KeeperTestSuite) TestMsgSendPacket() {
+	var path *ibctesting.Path
+	var msg *channeltypesv2.MsgSendPacket
+	var expectedPacket channeltypesv2.Packet
+
+	testCases := []struct {
+		name        string
+		malleate    func()
+		expError    error
+		shouldPanic bool
```

Comment:
This doesn't look like it is used anywhere, can we remove it?

Reply:
good catch, leftover from test that I removed